pulpcore/app/serializers/repository.py

```diff
@@ -435,12 +421,154 @@ class RepositoryVersionSerializer(ModelSerializer, NestedHyperlinkedModelSeriali
         lookup_field='number',
         parent_lookup_kwargs={'repository_pk': 'repository__pk'},
     )
+    content_hrefs = serializers.SerializerMethodField(
+        help_text=_('A mapping of the types of content in this version, and the HREF to view '
+                    'them.'),
+        read_only=True,
+    )
+    content_added_hrefs = serializers.SerializerMethodField(
+        help_text=_('A mapping of the types of content added in this version, and the HREF to '
+                    'view them.'),
+        read_only=True,
+    )
+    content_removed_hrefs = serializers.SerializerMethodField(
+        help_text=_('A mapping of the types of content removed from this version, and the HREF '
+                    'to view them.'),
+        read_only=True,
+    )
+    content_summary = serializers.SerializerMethodField(
+        help_text=_('A list of counts of each type of content in this version.'),
+        read_only=True,
+    )
+    content_added_summary = serializers.SerializerMethodField(
+        help_text=_('A list of counts of each type of content added in this version.'),
+        read_only=True,
+    )
+    content_removed_summary = serializers.SerializerMethodField(
+        help_text=_('A list of counts of each type of content removed in this version.'),
+        read_only=True,
+    )
+
+    def get_content_summary(self, obj):
+        """
+        The summary of contained content.
+
+        Returns:
+            dict: of {<type>: <count>}
+        """
+        annotated = obj.content.values('type').annotate(count=Count('type'))
```

Comment:
We should consider whether this should be made a method like ```added()``` and ```removed()``` to be more consistent, and also leave open the possibility of adding kwargs later.  But that can be a different PR.